### PR TITLE
Some updates:

### PR DESCRIPTION
**Pull request by piton4eg**
- use actual link to initializer
- change spread props to used props
- fix name of styles prop to butoonStyles

### CODE REVIEW

**Comment by piton4eg** [comment]
@pbernasconi Hi, could you look on this?

File: src/PlaidLink.js [hunk 1]
```diff
@@ -83,7 +83,7 @@ const PlaidLink = React.createClass({
     };
   },
   getScriptURL: function() {
-    return 'https://cdn.plaid.com/link/stable/link-initialize.js';
+    return 'https://cdn.plaid.com/link/v2/stable/link-initialize.js';
```
Comment (pbernasconi) [inline src/PlaidLink.js]: 👍 thanks for fixing this

File: src/PlaidLink.js [hunk 2]
```diff
@@ -71,7 +71,7 @@ const PlaidLink = React.createClass({
     buttonText: React.PropTypes.string,
 
     // Button Styles as an Object
-    styles: React.PropTypes.object,
+    buttonStyles: React.PropTypes.object,
```
Comment (pbernasconi) [inline src/PlaidLink.js]: I think we should just rename this to `style` so that it mimics the standard style element prop

**Comment by pbernasconi** [comment]
@piton4eg I added some feedback, thanks for this PR!

**Comment by piton4eg** [comment]
@pbernasconi updated ;)

**Pull request merged**